src/index.html

```diff
@@ -7,6 +7,130 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <ul class="calendar calendar--start-day--sat">
+      <li class="calendar__day">
+        <a href="#day1" class="calendar__link"></a>
+      </li>
```

Comment:
`
```suggestion
      <li class="calendar__day"></li>
```
You don't need content inside items.